integrations/ashby/nango.yaml

```diff
@@ -38,6 +38,72 @@ integrations:
                 endpoint:
                     method: POST
                     path: /notes
+            application-changeSource:
+                output: AshbyResponse
+                input: ChangeSource
+                description: |
+                    Action to change source of application.
+                endpoint:
+                    method: POST
+                    path: /application/source
+                    group: Application
+                scopes:
+                    - candidatesWrite
+            application-changeStage:
+                output: AshbyResponse
+                input: ChangeStage
+                description: |
+                    Action to change stage of an application.
+                endpoint:
+                    method: POST
+                    path: /application/stage
+                    group: Application
+                scopes:
+                    - candidatesWrite
+            application-update:
+                output: AshbyResponse
+                input: ChangeSource | ChangeStage | UpdateHistory
+                description: |
+                    Action to update an application stage.
+                endpoint:
+                    method: POST
```

Comment:
Even though the API call is a POST can we make this a PATCH to fit with under updates?

```
method: PATCH
path: /applications
```